### PR TITLE
JSON POST requests

### PR DESCRIPTION
**Pull request by dizcza**
Done:
* deserialize json into neo objects
* call elephant with the input arguments
* recursively serialize the output and send back in json
* demo

Concerns:
* serialization and de-serialization is the bottleneck. We'll come to this during the meeting with EBRAINS guys
* no support for classes. Elephant is becoming OOP more and more. We'll probably need to specify auxiliary functions in elephant-server to deal with each and every case where an object is created in elephant.
* the idea of moving the kwards under `data` key in json request dict has not proven to be ideal. Check out the demo notebook to see the usage - it gets complicated.

### CODE REVIEW

File: demo.ipynb [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "response = requests.post('http://localhost:5000/api/statistics/isi', json=dict(\n",
+    "    data=dict(spiketrain=[1,2,3,4,11]),\n",
+    "    units=dict(time='s')  # specify that all units are in secods\n",
```
Comment (babsey) [inline demo.ipynb]: secods -> seconds

File: README.md [hunk 2]
```diff
@@ -36,27 +35,22 @@ Or start with Docker
...
+
+Using curl in bash
 ```
-import requests
-requests.post('http://localhost:5000/api/statistics/isi', json={'spiketrain': [1,2,4,7,11]}).json()
+curl localhost:5000/api/<module>/<call>
 ```
 
-Use ElephantClientAPI object in Python (`ipython`)
+E.g. To see a list of calls in statistucs module
```
Comment (babsey) [inline README.md]: statistucs -> statistics

**Review by babsey — CHANGES_REQUESTED** [review]
I like the PR. You can find some minor typos in inline comments. 
Once you fixed it, I will approve the merge

**Comment by dizcza** [comment]
I suggest making "Squash and merge" pull request commits by default to keep the master clean of the intermediate commits.

Settings -> Options -> Merge Button -> untick "Allow merge commits" and leave "Allow squash merging" only.

This is a good practice for any project.

**Pull request merged**